### PR TITLE
Update dd-agent to the last stable version

### PR DESCRIPTION
**Pull request by kuzaxak**
Version 7 has a py3 which can introduce problems with our checks. Based on that decided to stay on version 6. 

[1]: https://hub.docker.com/r/datadog/agent/tags

### CODE REVIEW

**Comment by deiwin** [comment]
We only have two very short custom checks. Couldn't we update?

The [upgrade guide](https://docs.datadoghq.com/agent/guide/python-3/) seems pretty comprehensive and has examples of linters to find out if everything works as expected. There's also [a way](https://docs.datadoghq.com/agent/guide/agent-v6-python-3/?tab=hostagent) to check the compatibility with the existing 6.15 images.

**Comment by kuzaxak** [comment]
@deiwin Made an update to v7

**Comment by kuzaxak** [comment]
!build

File: Jenkinsfile [hunk 1]
```diff
@@ -21,7 +28,7 @@ withResultReporting(slackChannel: '#tm-inf') {
     stage('Scan image') {
       imageScanner.scan(image)
     }
-    if (BRANCH_NAME == MAIN_BRANCH) {
+    if (BRANCH_NAME == MAIN_BRANCH || isForcePublish) {
```
Comment (deiwin) [inline Jenkinsfile]: A comment in the PR in case of `isForcePublish` would be helpful

**Comment by kuzaxak** [comment]
!build

**Comment by sm-deployer** [comment]
Built and published `7.16.1-f551b6c`

**Comment by kuzaxak** [comment]
!build

**Comment by sm-deployer** [comment]
Built and published `7.16.1-29f4170`

**Comment by kuzaxak** [comment]
!build

**Comment by sm-deployer** [comment]
Built and published `7.16.1-d002e6f`

**Pull request merged**